core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/layout/utils/ImageComparison.java

```diff
@@ -79,24 +80,26 @@ public static ImageComparisonResult compare(final BufferedImage pattern, final B
             img1Pixels, img2Pixels);
 
     // Sets not covered areas as error (red)
-    fastMarkOuterAreaAsError(minWidth, minHeight, widthDifference, heightDifference,
+    differenceCounter += fastMarkOuterAreaAsError(minWidth, minHeight, widthDifference, heightDifference,
```

Comment:
I'm wondering - but it is probably a good idea to count outer area as errors.
Please update comment above in line `82`.